src/dfx/assets/build.rs

```diff
@@ -34,9 +34,9 @@ fn calculate_hash_of_inputs(project_root_path: &Path) -> String {
     hex::encode(sha256.finish())
 }
 
-fn find_assets() -> PathBuf {
-    println!("cargo:rerun-if-env-changed=DFX_ASSETS");
-    if let Ok(a) = env::var("DFX_ASSETS") {
+fn find_frontend() -> PathBuf {
```

Comment:
Looks like all of the changes in this file should be reverted
```suggestion
fn find_assets() -> PathBuf {
```